README.md

```diff
@@ -1,26 +1,78 @@
+
+```js
+
+module.exports = {
+  debug: true,
+  discord: {...}
+}
+
+```
+
+### Global ESlint Install
+
+ESlint has the ability to be install globally (allows for cmd line useage, which is helpful for coding), however since we use a few plugins for ESlint that usage won't work out of the box, so you'll need to install them. We've had luck using
+
+```batch
 
-However, if you do want to run an instance, you need the following:
+npm i -g eslint eslint-plugin-standard@latest eslint-plugin-promise@latest eslint-plugin-import eslint-config-standard@latest eslint-plugin-node@latest
+
+```
+
+### Coming soon!
+
+Tests! You can figure out if you broke something sending us a super cool feature with a simple `npm test`!
+
+---
+
+## Selfhosting
+
+We don't (fully) support running your own copy of our bot, as its highly customized for usage on the Discord Feedback server. This doesn't mean you can't run it, just that we don't have the time to help you make it run (you know, with making this bot and all).
+
+Now, if you do want to run an instance, you need the following:
 
 - UserVoice API keys, these **must** be marked as "Trusted" inside the admin panel.
 - Discord bot account
 - [RethinkDB](https://www.rethinkdb.com)
 - A Sentry DSN
+- The bot to be installed (duh) `npm i`
 - Discord server (duh)
   - Server **must** have channels named `bot-log`, `admin-queue` and `bot-error`, and the bot **must** have write access to these channels.
-  
-## Config creation
-Config creation is pretty straightforward, all values given in the example config file (`config.example.js`) are placeholders, replace them with your own data and save the file as `config.js` in the project root.
 
-## Database initialization
-Run `npm run-script dbcreate` in the project root to initialize RethinkDB with the required tables.
+### Config creation
+
+Config creation is pretty straightforward, everything you need is provided for you in the example config file (`config.example.js`), all the values in there are placeholders, just replace them with your own data and then save the file as `config.js` in the project root.
+
+### Database initialization
+
+We use a database to track things, so you gotta get RethinkDB setup with the tables and whatnot the bot needs. Luckly for you we made it super simple with out dbcreate script™. Just run the command `npm run-script dbcreate` in the project root (**While RethinkDB server is running!**) in order initialize RethinkDB with all those tables that is needed, and you're good to go!
+
+### Running the Bot
 
-## Starting
-Run `npm start`, please note there will be little to no console output, this program is highly reliant on [Sentry](https://sentry.io) for error reporting.
+Starting is also very simple, just run `npm start`. Do note there will be little to no console output, this program is highly reliant on [Sentry](https://sentry.io) for error reporting.
```

Comment:
We don't actually use `npm start` in production though 🤔

Reply:
While true, its the same of above it being intended more for people wanting to contribute.